[2023-05-20 16:16:14,973] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2023-05-20 16:17:45,775] 	ERROR 	115 	pair_data.py 	<module>() 	Error occurred during Pair Data process. Error: name 'pairdata_full' is not defined
